pages/linux/spectre-meltdown-checker.md

```diff
@@ -7,7 +7,7 @@
 
 `sudo spectre-meltdown-checker`
 
-- Check the currently running kernel and show an explanation of the actions to take in order to mitigate a vulnerability:
+- Check the currently running kernel and show an explanation of the actions to take to mitigate a vulnerability:
```

Comment:
I still remember when I wrote this line! 😅

Reply:
Reminds me whenever I'm at work looking at code. 
I see something I hate, do `git blame` to see who wrote it, and look down in shame when it reveals it was me. ^-^'